ktor-client/ktor-client-core/common/test/DefaultRequestTest.kt

```diff
@@ -111,14 +112,34 @@ class DefaultRequestTest {
         val defaultUrl = Url(URLBuilder.Companion.origin)
         if (defaultUrl.port == defaultUrl.protocol.defaultPort) {
             assertEquals("https://localhost", client.get {}.bodyAsText())
-            assertEquals("ws://other.host:443/", client.get("ws://other.host/").bodyAsText())
```

Comment:
could you update these checks to present values instead of removing them?

Reply:
It was not dropped, but moved outside the `if`.